libs/mocks/WordPressMocks/src/main/assets/mocks/mappings/wpcom/auth/is-available_email_not_available.json

```diff
@@ -4,7 +4,7 @@
         "urlPath": "/is-available/email/",
         "queryParameters": {
             "q": {
-                "matches": "^(?!new@example.com).*$"
```

Comment:
The intention of this regex is for everything except the user we use to sign up to give this response. That was the purpose of `new@example.com` here. I think the `.*` is still needed, but of course `new@example.com` should be updated to whatever email we decide on.

Reply:
Ahh I see I misunderstood this, thanks for clarifying! Is this needed for the Login Tests? I'm wondering why it was there before before the Signup Tests or was it just in preparation for that?

Reply:
Yeah, it was in anticipation of sign up 😄